<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="placeholderText">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="roastLevelEdit">
     <property name="placeholderText">
      <string>Степень обжарки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="typeEdit">
     <property name="placeholderText">
      <string>Тип (молотый/в зернах)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="tasteEdit">
     <property name="placeholderText">
      <string>Описание вкуса</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="priceEdit">
     <property name="placeholderText">
      <string>Цена</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="packageSizeEdit">
     <property name="placeholderText">
      <string>Объем упаковки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>